### ISSUE TITLE
Warning: Failed prop type: DropzoneArea: prop type `initialFiles` is invalid

### ISSUE THREAD

**Issue by Tassfighter** [Yuvaleros/material-ui-dropzone#78]
After moving from v2.4.1 to v2.4.3

My app throws:

> Warning: Failed prop type: DropzoneArea: prop type `initialFiles` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.

**Comment by juancarloselorriaga** [comment]
Same issue here.

**Comment by AhmedLight92** [comment]
same issue here

**Comment by alexandredavi** [comment]
+1

**Comment by Tassfighter** [comment]
src/DropzoneArea.js

`initialFiles: PropTypes.arrayOf[PropTypes.string],`

should be:

`initialFiles: PropTypes.arrayOf(PropTypes.string),`?

see [react typechecking](https://reactjs.org/docs/typechecking-with-proptypes.html) arrayOf example:
```
// An array of a certain type
optionalArrayOf: PropTypes.arrayOf(PropTypes.number),
```

**Comment by ksundeen** [comment]
Yes, this works @Tassfighter . I saved a patch in the Dropzone.propTypes.

**Comment by Yuvaleros** [comment]
Fixed issue in v2.4.5
